context.lisp

```diff
@@ -0,0 +1,103 @@
+(in-package :cl+ssl)
+
+(define-condition verify-location-not-found-error (ssl-error)
+  ((location :initarg :location))
+  (:documentation "Unable to find verify locations")
+  (:report (lambda (condition stream)
+              (format stream "Unable to find verify location. Path: ~A" (slot-value condition 'location)))))
+
+(defun validate-verify-location (location)
+  (handler-case
+      (cond
+        ((uiop:file-exists-p location)
+         (values location t))
```

Comment:
It's a dependency of ASDF, but not in all ASDF versions. UIOP exists as a separate module and is available in Quicklisp.